src/app/rpmostree-compose-builtin-tree.c

```diff
@@ -1012,6 +1031,13 @@ impl_compose_tree (const char      *treefile_pathstr,
       }
   }
 
+  if (!rpmostree_rootfs_postprocess_common (self->rootfs_dfd, cancellable, error))
+    return EXIT_FAILURE;
+  if (!rpmostree_postprocess_final (self->rootfs_dfd, self->treefile,
```

Comment:
Hey, may be a silly question here as I am kinda confused about the postprocess part. Hopefully you won't mind :). I kinda got a few doubts. 
1:  It seems like before, we are able to run `rpmostree_treefile_postprocessing()` after we install packages into the rootfs. But now for the commit and postprocess case, we might not even install any packages. So wondering what will the postprocess be in the commit case look like, what might it achieve?

Reply:
Basically in the test flow I'm doing, the `yum install` type stuff lives in a `Dockerfile`.  But implementing e.g. the "convert /var to tmpfiles.d" is quite nontrivial and I don't think we should duplicate it in a `Dockerfile`.  Same thing with the "checksum the kernel/initramfs" etc. that ostree requires.